client/components/async-load/index.jsx

```diff
@@ -6,8 +6,13 @@ import { omit } from 'lodash';
 
 export default class AsyncLoad extends Component {
 	static propTypes = {
+		noPlaceholder: PropTypes.bool,
+		placeholder: PropTypes.node,
 		require: PropTypes.func.isRequired,
-		placeholder: PropTypes.node
+	};
+
+	static defaultProps = {
+		noPlaceholder: false,
```

Comment:
> couldn't we detect falsiness on placeholder and skip the boolean flag?

Right now, if you don't provide an explicit placeholder, we use a default placeholder. I _think_ it's ok to switch to requiring folks who want a placeholder to provide one, but it'll be a bigger change.